Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    r7insight (3.0.0)
```

Comment:
5 patch version increment...?

Reply:
i ran `make bump-patch` 5 times. the first 3 runs gave me some errors so i had to re-run

Reply:
what errors? if they can be fixed thy should. then just revert to master and run it once, versions should be sequential

Reply:
many. like cannot run make(no rvm), missing bundler bump, wrong ruby version, etc. i had to fix one by one(the errors were not shown in one run)

Reply:
if the number shouldn't skip, shall i just manually modify the version to `3.0.1`?

Reply:
Hm right, I'd prefer actually running `make bump-patch` since we're making sure we're not missing anything. Now that it works, why not just revert the commits, or rebase onto master and run `bump-patch`?

Reply:
yes. they're just version numbers. it didn't modify anything else in the repo.